kinto/core/storage/postgresql/__init__.py

```diff
@@ -420,7 +420,7 @@ def delete(
              WHERE id = :object_id
                AND parent_id = :parent_id
                AND resource_name = :resource_name
-               AND deleted = FALSE
+               AND NOT deleted
```

Comment:
Is this just cleaning up whilst-you're-at-it?

Also, why is it not using `core.storage.DEFAULT_DELETED_FIELD`?

Reply:
> Is this just cleaning up whilst-you're-at-it?

yes

> Also, why is it not using core.storage.DEFAULT_DELETED_FIELD?

hmm, good point, but the column is hard coded in the schema.sql file anyway